Adding some history and motivation to hubDocs

micokoch opened the pull request:
Adding info from Hubverse intro presentation to hubDocs - issue #52 in order to provide higher level background information.

nickreich reviewed (changes requested):
Thanks for this detailed PR that adds a ton of useful information! I have a few minor requests:

- [x] the image added under key definitions feels a little bit out of place. I like the idea of adding it here, but maybe it should be under its own header (e.g., it's not really a key definition), so that the list of key definitions doesn't just abruptly follow the figure with no explanation.
- [ ] Could you add a citation to the AJPH paper on the history page, which also catalogues some of these same points, and serves as sort of the source of the history of hubs figure (although the figure in that paper is a bit of an older version): https://ajph.aphapublications.org/doi/full/10.2105/AJPH.2022.306831

micokoch commented:
Hi @nickreich . I addressed all your comments/requests, except adding the `json` tag, for reasons explained below your comment. Let me know if you want me to add it anyway and if you need any additional changes. Thanks!

nickreich reviewed (approved):
Thanks, @micokoch ! LGTM.
